diff --git a/apps/dokploy/server/server.ts b/apps/dokploy/server/server.ts
index eaf56269..8a1805e6 100644
--- a/apps/dokploy/server/server.ts
+++ b/apps/dokploy/server/server.ts
@@ -60,6 +60,8 @@ void app.prepare().then(async () => {
 
 		if (IS_CLOUD && process.env.NODE_ENV === "production") {
 			await migration();
+			// PATCHED: Initialize schedules for remote servers (requires min 1 instance)
+			await initSchedules();
 		}
 
 		server.listen(PORT, HOST);
